R/ChromBackend.R

```diff
@@ -219,6 +241,16 @@
 #'   length equal `length(object)`. Note that missing values (`NA_character_`)
 #'   are not supported for `dataStorage()`.
 #'
+#' - `filterChromData()`: filters any numerical chromatographic data variables
+#'   the based on the provided numerical `ranges`. The method should return a
+#'   `ChromBackend` object with the chromatograms that match the condition. This
+#'   function will results in an object with less chromatogram than the original.
+#'
+#' - `filterPeaksData()`: filters any numerical peaks data variables based on
+#'   the provided numerical `ranges`. The method should return a `ChromBackend`
+#'   object with the chromatograms that match the condition. This function will
+#'   results in an object with the same number of chromatogram as the original.
```

Comment:
Maybe add *... as the original, but eventually less retention time - intensity value pairs.*. Otherwise it might not be clear what is actually filtered.